pkg/api/server/register_images.go

```diff
@@ -726,6 +726,11 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//    type: string
 	//    description: Allows for pushing the image to a different destination than the image refers to.
 	//  - in: query
+	//    name: forceCompressionFormat
+	//    description: Use the specified compression algorithm if the destination contains a differently-compressed variant already.
```

Comment:
I still have issues reading this sentence.  I can be interpreted that we'll only compress "if the destination contains a differently-compressed variant already".  So we won't compress if there's no variant at all on the registry?

I suggest "Enforce compressing the layers with the specified --compression and do not reuse differently compressed blobs on the registry."  I find that easier to read/understand.